tests/rbr/MakeRegionsTest.java

```diff
@@ -0,0 +1,76 @@
+package rbr;
+
+import java.util.*;
+import org.junit.*;
+import util.*;
+
+public class MakeRegionsTest {
+
+    Set<Character> ip = new HashSet<>();
+    Set<Character> op = new HashSet<>();
+    Set<Vertex> dst = new HashSet<>();
+    private RBR rbr;
+
+    @Before
+    public void setUp() {
+        Graph graph = RegularGraphBuilder.generateGraph(4,4);
+        rbr = new RBR(graph);
+        ip.add('I');
```

Comment:
Rename the variables to something less coded and that reflect multiplicity: 'ip' to 'inputPorts', 'op' to 'outputPorts', 'dst' to 'destinations' and so forth.